<launch>
<!-- Launch file for Livox AVIA LiDAR -->

  <arg name="rviz" default="true" />

  <node 
		pkg="std_loop" 
		type="wild_place" 
		name="wild_place" 
		output="screen" 
	/>
  <arg name="arg1" default="0.01"/>
  <param name="icp_threshold" value="$(arg arg1)" />
  <param name="calc_gt_enable" type="bool" value = "true"/>
  <param name="data_name" type="string" value = "karawatha04"/>
  <param name="setting_path" type="string" value = "$(find std_loop)/config/config_wild_place.yaml"/>
  <param name="pcds_file" type="string" value = "/home/ycj/data/wild-places/Karawatha/04/Clouds/"/>
  <param name="pose_file" type="string" value = "/home/ycj/data/wild-places/Karawatha/04/poses_sample.txt"/>
  <param name="loop_gt_file" type="string" value = "/home/ycj/matlab_code/loop_detection/gt/karawatha04_gt.txt"/>

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find std_loop)/rviz_cfg/wild_place.rviz" />
  </group>

</launch>